Makefile

```diff
@@ -9,6 +9,16 @@ install-dependencies:
 	npm ci --prefer-offline --include=dev
 	npx playwright install
 
+# Install additional test dependencies to run VS Code testing in headless mode
+# ref: https://code.visualstudio.com/api/working-with-extensions/continuous-integration#github-actions
+.PHONY: install-test-dependencies
+install-test-dependencies:
+	@echo "Installing test dependencies for $(shell uname -s)"
```

Comment:
Should we maybe add a comment here saying that this target is not used for the Windows agent?

Reply:
How's https://github.com/confluentinc/vscode/pull/453/commits/77b200f0475bc8c80d45e9cf646be538e7004af0?